docker/rabbit-mq/Dockerfile

```diff
@@ -1 +1 @@
-FROM rabbitmq:3.8-management-alpine
+FROM rabbitmq:3.8-alpine
```

Comment:
The management ui has already been removed. However, the overview dashboard isn't yet a replacement for the whole management interface. A dashboard for queues and possibly exchanges still needs to be added.

Reply:
`/metrics/per-objects` don't seem to be resource efficient and should be used sparingly. Let's try it without. It might be enough to find bottlenecks during load tests. If not, more detailed metrics can still be added later. With the latest commit, prometheus is using `/metrics`, not `/metrics/per-objects`.

Reply:
Done. Not a separate dashboard, but included in the `RabbitMQ Overview` in the `Queues` section.